docs/hooks/hooks.md

```diff
@@ -6,4 +6,1335 @@ sidebar_position: 4
 
 Scaffold-ETH 2 provides a collection of custom React hooks designed to simplify interactions with your deployed smart contracts. These hooks are wrappers around `wagmi`, automatically loading the necessary contract ABI and address. These hooks offer a simplified and streamlined interface for interacting with your smart contracts.
 
-If you need to interact with external contracts, you can use `wagmi` directly, or add external contract data to your `deployedContracts.ts` file.
+If you need to interact with external contracts you can use `wagmi` directly, or you can add external contract data to your `externalContracts.ts` file, which would let you use Scaffold-ETH 2 hooks with autocompletion to interact with them.
+
+To do that, you will have to add the contract `address` and `abi` for every contract you define in your `externalContracts.ts` file. Need to register this info for all the different network IDs where you want to use them. You will have to change the `targetNetwork` of your `scaffold.config.ts` file to your desired network to be able to use the hooks with autocompletion.
+
+This is the structure of `externalContracts` object:
+
+```ts
+const externalContracts = {
+  1: {
+    DAI: {
+      address: "0x...",
+      abi: [...],
+    },
+    WETH: {
+      address: "0x...",
+      abi: [...],
+    },
+  5: {
+    DAI: {
+      address: "0x...",
+      abi: [...],
+    },
+    USDT: {
+      address: "0x...",
+      abi: [...],
+    },
+} as const;
+```
+
+<details>
```

Comment:
Even if this is folded, it seems like too much, and I think the example above is enough.